src/public/manifest.json

```diff
@@ -11,16 +11,12 @@
     "activeTab",
     "cookies"
   ],
-  "content_scripts": [
-    {
-      "matches": [
-        "<all_urls>"
-      ],
-      "js": [
-        "content.js"
-      ]
-    }
-  ],
+  "background": {
+    "scripts": [
+      "browser-polyfill.min.js",
+      "background.js"
```

Comment:
The content_script in chrome doesn't support cors request. So, move it to background script.